### PR TITLE
Expose FS.resetIdleTimer

### PR DESCRIPTION
**Pull request by mbrubeck**
Requires version 1.14.0 of the FullStory SDK for mobile apps.

### CODE REVIEW

**Comment by JoshMiers-FS** [comment]
This will also need a way to force a minimum FS version, or else there will unexplained compilation errors.

**Comment by mbrubeck** [comment]
> This will also need a way to force a minimum FS version, or else there will unexplained compilation errors.

I added a check for this in the Android build script, but I haven't found a way to do it on iOS.

**Comment by mbrubeck** [comment]
Pushed an additional change to require FullStory 1.14 on iOS.

File: android/build.gradle [hunk 1]
```diff
@@ -70,6 +70,18 @@ if (fsDetectedVersion == null) {
...
+def versionRegex = /(?<major>\d+)\.(?<minor>\d+)\.(?<patch>\d+)/
+def match = fsDetectedVersion =~ versionRegex
+if (match.matches()) {
+    def major = match.group('major') as Integer
+    def minor = match.group('minor') as Integer
+    if (major < 1 || (major == 1 && minor < 14)) {
+        throw new GradleException("FullStory SDK version 1.14.0 or later is required, but version $fsDetectedVersion was detected. Please update to a newer version of FullStory.")
+    }
+} else {
+    throw new GradleException('Unable to determine FullStory version, please verify your root build file is properly configured')
```
Comment (JoshMiers-FS) [inline android/build.gradle:82]: Maybe just log that we cannot detect the FS version

Comment (mbrubeck) [reply android/build.gradle]: Done.

**Review by JoshMiers-FS — APPROVED** [review]
LGTM!

File: fullstory_react-native.podspec [hunk 2]
```diff
@@ -20,7 +20,7 @@ Pod::Spec.new do |s|
   s.requires_arc = true
 
   s.dependency "React"
-  s.dependency "FullStory"
+  s.dependency "FullStory", "~> 1.14"
```
Comment (prashanthsadasivan-fullstory) [inline fullstory_react-native.podspec:23]: @mbrubeck should this be changed? seems like it's hard coding it to our 1.14 SDK release.

Comment (prashanthsadasivan-fullstory) [reply fullstory_react-native.podspec:23]: Oh my bad, i missed the last review, we want this to be declared. ignore this!

**Pull request merged**